player/command.c

```diff
@@ -7234,8 +7234,10 @@ static void command_event(struct MPContext *mpctx, int event, void *arg)
         ctx->marked_permanent = false;
     }
 
-    if (event == MPV_EVENT_PLAYBACK_RESTART)
+    if (event == MPV_EVENT_PLAYBACK_RESTART) {
         ctx->last_seek_time = mp_time_sec();
+        run_command_opts(mpctx);
```

Comment:
This makes it run after every seek. Was that intended?

Reply:
Oh right, where it currently is is functionally the equivalent. If it can be moved somewhere else, that would be preferred but not a huge deal.